inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/convolution.cpp

```diff
@@ -102,6 +102,33 @@ INSTANTIATE_TEST_CASE_P(
         ::testing::Values(CommonTestUtils::DEVICE_CPU)),
     ConvolutionLayerTest::getTestCaseName);
 
+namespace specificWeightLayout {
```

Comment:
Could you please add more detailed description why we need this test into the comment?

Reply:
description added